client/src/pages/AddFood.js

```diff
@@ -35,46 +38,45 @@ function AddFood() {
   });
 
   function handleForm(e) {
-    e.preventDefault();
     mutation.mutate(url, { title: "title", img: "img", author: "author" });
```

Comment:
This seems like a place we should focus in our next 1:1.  Perhaps a session where I drive and you ask questions?